tests/testerina-integration-test/src/test/resources/project-based-tests/object-mocking-tests2/Dependencies.toml

```diff
@@ -0,0 +1,38 @@
+# AUTO-GENERATED FILE. DO NOT MODIFY.
```

Comment:
Do we need `.gitignore`, `Dependencies.toml` files to be comited here?

Reply:
I'll remove those files.